test/unittests/test_huffman.cpp

```diff
@@ -0,0 +1,251 @@
+#include "core/truthTable/truth_table.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace dd::literals;
+using namespace syrec;
+
+class TruthTableHuff: public testing::TestWithParam<std::string> {
+protected:
+    dd::QubitCount                 nqubits = 3U;
+    TruthTable                     tt{};
+    std::unique_ptr<dd::Package<>> dd;
+
+    TruthTable::Cube::Value emptyVal;
+
+    TruthTable::Cube c00{};
+    TruthTable::Cube c01{};
+    TruthTable::Cube c10{};
+    TruthTable::Cube c11{};
+
+    TruthTable::Cube c000{};
+    TruthTable::Cube c001{};
+    TruthTable::Cube c010{};
+    TruthTable::Cube c011{};
+    TruthTable::Cube c100{};
+    TruthTable::Cube c101{};
+    TruthTable::Cube c110{};
+    TruthTable::Cube c111{};
+
+    TruthTable::Cube c_11{};
+    TruthTable::Cube c1_0{};
+
+    TruthTable::Cube c0_{};
+    TruthTable::Cube c1_{};
+
+    TruthTable::Cube c0__{};
+    TruthTable::Cube c1__{};
+
+    TruthTable::Cube c10_{};
```

Comment:
similar to the other two test files, this should be refactored to make use of the PLA parser.

Reply:
I have removed the most of the cubes, however, I have kept a few for checking whether particular cube instances are added or not (outputs of a Pla aren't supposed to have don't cares?).